webpack.common.js

```diff
@@ -35,7 +35,15 @@ module.exports = (env = {}) => ({
           options: {
             plugins: [
               // TODO: Add necessary remark plugins
-              require('remark-autolink-headings'),
+              require('./tip'),
+              require('remark-slug'),
+              [
+                require('remark-autolink-headings'),
+                {
+                  behaviour: 'append'
+                }
+              ],
+              require('remark-html'),
```

Comment:
Can you separate these changes to another PR? They seem unrelated to the google-analytics fix.